test/connection_switching_test.rb

```diff
@@ -17,6 +17,41 @@ def clear_connection_pool
     require 'models'
   end
 
+  after do
+    ActiveRecord::Base.default_shard = nil
+  end
```

Comment:
This is to  ensure that after setting `default_shard=` in any test that it gets set back to `nil` so it doesn't leak over into other tests.